src/test/compile-fail/dupe-symbols-2.rs

```diff
@@ -11,13 +11,13 @@
 #![crate_type="rlib"]
 #![allow(warnings)]
 
-mod a {
+pub mod a {
     #[no_mangle]
     pub extern fn fail() {
     }
 }
 
-mod b {
+pub mod b {
     #[no_mangle]
     pub extern fn fail() {
```

Comment:
Shouldn't private `no_mangle` functions always be compiled in?

Reply:
Seems like it would be good if that were true.